DSharpPlus/Entities/Thread/DiscordThreadChannel.cs

```diff
@@ -0,0 +1,690 @@
+
+        /// <summary>
+        /// Gets ID of the news or text channel that contains this thread.
+        /// </summary>
+        [JsonProperty("parent_id", NullValueHandling = NullValueHandling.Ignore)]
+        public ulong ParentId { get; internal set; }
+
+        /// <summary>
+        /// Gets the name of this thread.
+        /// </summary>
+        [JsonProperty("name", NullValueHandling = NullValueHandling.Ignore)]
+        public string Name { get; internal set; }
+
+        /// <summary>
+        /// Gets the type of this thread.
+        /// </summary>
+        [JsonProperty("type", NullValueHandling = NullValueHandling.Ignore)]
+        public ChannelType Type { get; internal set; }
+
+        /// <summary>
+        /// Gets whether this thread is private.
+        /// </summary>
+        [JsonIgnore]
+        public bool IsPrivate
+            => this.Type == ChannelType.PrivateThread;
+
+        /// <summary>
+        /// Gets the ID of the last message sent in this thread.
+        /// </summary>
+        [JsonProperty("last_message_id", NullValueHandling = NullValueHandling.Ignore)]
+        public ulong? LastMessageId { get; internal set; }
+
+        /// <summary>
+        /// <para>Gets the slow mode delay configured for this thread.</para>
+        /// <para>All bots, as well as users with <see cref="Permissions.ManageChannels"/> or <see cref="Permissions.ManageMessages"/> permissions in the channel are exempt from slow mode.</para>
+        /// </summary>
+        [JsonProperty("rate_limit_per_user", NullValueHandling = NullValueHandling.Ignore)]
+        public int? PerUserRateLimit { get; internal set; }
+
+        /// <summary>
+        /// Gets an approximate count of messages in a thread, stops counting at 50.
+        /// </summary>
+        [JsonProperty("message_count", NullValueHandling = NullValueHandling.Ignore)]
+        public int? MessageCount { get; internal set; }
+
+        /// <summary>
+        /// Gets an approximate count of users in a thread, stops counting at 50.
+        /// </summary>
+        [JsonProperty("member_count", NullValueHandling = NullValueHandling.Ignore)]
+        public int? MemberCount { get; internal set; }
+
+        /// <summary>
+        /// Gets when the last pinned message was pinned in this thread.
+        /// </summary>
+        [JsonIgnore]
+        public DateTimeOffset? LastPinTimestamp
+            => !string.IsNullOrWhiteSpace(this.LastPinTimestampRaw) && DateTimeOffset.TryParse(this.LastPinTimestampRaw, CultureInfo.InvariantCulture, DateTimeStyles.None, out var dto) ?
+                dto : null;
+
+        /// <summary>
+        /// Gets when the last pinned message was pinned in this thread as raw string.
+        /// </summary>
+        [JsonProperty("last_pin_timestamp", NullValueHandling = NullValueHandling.Ignore)]
+        internal string LastPinTimestampRaw { get; set; }
+
+        /// <summary>
+        /// Gets the threads metadata.
```

Comment:
Read: redundant docs. "Gets metadata about the thread, e.g. [...]" something like that.